### DISCUSSION TITLE
Welcome to cpp_weekly Discussions!

### DISCUSSION THREAD

**Discussion by lefticus** [lefticus/cpp_weekly#142] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sumakana** [comment]
Wassup chief! I have some ideas which arise from the need to translate MATLAB m code without the use of its built-in C/C++ coder.

Could you do a review of CPP math libraries, along with their strengths and weaknesses?
Also, I would appreciate tips and tricks when adapting source code to MISRA C/C++ standards.
Differences between AUTOSAR and MISRA C/C++ standards.
Thank you for your time, keep up the awesome work. Respect.

**Comment by lefticus** [comment]
Can you make issues for these 2 different ideas so that we can get community upvotes on them? 

https://github.com/lefticus/cpp_weekly/issues

**Comment by WotJProject** [comment]
I'm a C++ game developer, building my own game engine from scratch (with Vulkan, if that counts). I've really enjoyed your YouTube channel, and it fits really well with my "C with extra features" style of coding. I strongly dislike inheritance hierarchies, virtual functions, and needless abstraction, but I really love classes, polymorphism and other OOP features that make C++ great to work with!

You asked me to post my question about pack indexing to here, but I wasn't sure which discussion category to put it in, so I put it in Q&A. It's an interesting problem for an interesting feature, and I'm enjoying discussing it with the other community member who's commenting on it.
